### PR TITLE
wip

### PR DESCRIPTION
**Pull request by corneliusroemer**
https://preview-virus2.ppx.bio

### CODE REVIEW

**Comment by emmahodcroft** [comment]
Some checklist items:

[ ] Move example data from private repo to the public example data repo (only files that aren't designed to error)
[ ] Blog post (news item)
[ ] Tweet thread
[ ] Check demo.pathoplexus.org still works
[ ] Upload any held EB resolutions relating to the launch

(Please add more!)

**Comment by anna-parker** [comment]
Maybe we could do https://github.com/loculus-project/loculus/issues/1797 soonish - the deletion and insertion lists are really long for a lot of mpox sequences

**Review by theosanderson — APPROVED** [review]
Looks reasonable to me! One unexpected (?) comment

File: loculus_values/values.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ runDevelopmentMainDatabase: true
 bannerMessage: This is a preview instance. Do not rely on this data for real analyses.
 gitHubEditLink: https://github.com/pathoplexus/pathoplexus/edit/main/monorepo/website/
 runDevelopmentKeycloakDatabase: true
+developmentDatabasePersistence: true # TODO: revert before merging
```
Comment (corneliusroemer) [inline loculus_values/values.yaml:9]:
```suggestion
developmentDatabasePersistence: false
```

**Pull request merged**